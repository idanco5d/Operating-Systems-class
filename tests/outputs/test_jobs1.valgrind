==8528== Memcheck, a memory error detector
==8528== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==8528== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==8528== Command: /mnt/c/Users/illus/os/hw/1/smash
==8528== Parent PID: 8522
==8528== 
==8528== 
==8528== FILE DESCRIPTORS: 4 open (3 std) at exit.
==8528== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_jobs1.valgrind
==8528==    <inherited from parent>
==8528== 
==8528== 
==8528== HEAP SUMMARY:
==8528==     in use at exit: 0 bytes in 0 blocks
==8528==   total heap usage: 60 allocs, 60 frees, 81,169 bytes allocated
==8528== 
==8528== All heap blocks were freed -- no leaks are possible
==8528== 
==8528== For lists of detected and suppressed errors, rerun with: -s
==8528== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
