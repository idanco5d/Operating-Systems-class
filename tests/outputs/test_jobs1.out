smash> smash> smash> smash> smash> smash> [1] sleep 100& : 9561 0 secs
smash> smash: sending SIGKILL signal to 1 jobs:
9561: sleep 100&
